metadata/src/test/java/org/apache/kafka/metadata/PartitionRegistrationTest.java

```diff
@@ -283,7 +283,7 @@ private static Stream<Arguments> metadataVersionsForTestPartitionRegistration()
         return Stream.of(
             MetadataVersion.IBP_3_7_IV1,
             MetadataVersion.IBP_3_7_IV2,
-            MetadataVersion.IBP_3_8_IV0
+            MetadataVersion.IBP_3_8_IV1
```

Comment:
Should we make the same change in PartitionChangeBuilderTest?

Reply:
My miss, updated in the next commit

Reply:
This corresponds to ELR and needs to be changed to BP_4_0_IV0 like PartitionChangeBuilderTest.java.  Ditto below.

@CalvinConfluent : Could you confirm that? Also, could you check if there is any other change needed for ELR in this PR?

Reply:
QuorumControllerTest.java: testUncleanShutdownBroker